src/python/doc/datasets_generators.rst

```diff
@@ -0,0 +1,97 @@
+
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+===========================
+Datasets generators manual
+===========================
+
+We provide the generation of different customizable datasets to use as inputs for Gudhi complexes and data structures.
+
+
+Points generators
+------------------
+
+Points on sphere
+^^^^^^^^^^^^^^^^
+
+The module **_points** enables the generation of random i.i.d. points uniformly on a (d-1)-sphere in :math:`R^d`.
```

Comment:
It is a bit strange to say this is provided by the module, and only see the name of the actual function in the example.

Reply:
Hmm, do you mean to get to the function right away without mentioning the module ? We only see the actual function in the example because the section is about the sphere.